src/router/bundle/bundleParser.test.ts

```diff
@@ -1331,5 +1331,55 @@ describe('parseResource', () => {
                 );
             }
         });
+
+        test('Do not consider "referenceSeq" as a "reference". Only resource fields explicitly named "reference" should be considered a reference', async () => {

```

Comment:
Without the code change in `bundleParser.ts` this test will fail with this error
`Error: This entry's reference is not recognized. Entry's reference is: http://www.ncbi.nlm.nih.gov/nuccore . Valid format includes "<url>/resourceType/id" or "<urn:uuid:|urn:oid:><id>
`

With the change in `bundleParser.ts`, this test passes.